insights/archive/repository/base_archives/rhel7/insights_commands/rhv-log-collector-analyzer_--json

```diff
@@ -0,0 +1,188 @@
+{
```

Comment:
@dougsland it is not necessary to add this to the archive template.  It is used to test the archive code.

Reply:
Thanks, removing.